src/content/site/faq.md

```diff
@@ -31,3 +31,7 @@ This is harmless fun, if you just accept it, laugh and move on, no problem.
 If you get annoyed that you don't have your name and make a big deal out of it, you're gonna have a bad time. Everyone, including the staff, follows nickname trends and jokes.
 
 5.8% of people (yes, we have actually polled for this) dislike it, and much less actually leave because of it. Don't be the sour apple if you get nicknamed something funny.
+
+## How do I join The Programmer's Hangout GitHub organisation?
```

Comment:
@veksen was this an issue that we asked for? I am just unsure about issues that people create on here and then commit to. Especially during hacktoberfest.

Reply:
no, but it's valid content! going through